db/schema.rb

```diff
@@ -143,10 +143,10 @@
     t.boolean "open_banking_consent"
     t.datetime "open_banking_consent_choice_at"
     t.string "own_home"
-    t.decimal "percentage_home"
     t.decimal "property_value", precision: 10, scale: 2
-    t.decimal "outstanding_mortgage_amount"
+    t.decimal "percentage_home"
     t.string "shared_ownership"
+    t.decimal "outstanding_mortgage_amount"
```

Comment:
I think this file shouldn't be modified.
If you revert it to the version in master and use `rails db:schema:load` that might fix your db (not sure)